src/selectors/user.js

```diff
@@ -25,6 +25,10 @@ import type { RootReducerState } from 'reducers/rootReducer';
 import type { User } from 'models/User';
 import type { Selector } from 'selectors';
 
+import { useRootSelector } from './selectors';
+
+export const useUser = () => useRootSelector((root) => root.user.data);
+
 export const userSelector = ({ user: { data } }: RootReducerState): User => data;
```

Comment:
Do we need a custom hook where the selector below would suffice? When reading a component, it's clearer what to expect from `useRootSelector(userSelector)` than `useUser()`.

Reply:
IMO it make sense for _most common_ use cases. I agree that somewhat duplicates `userSelector`, but on the other hand it's simplier to use in FCs than `userSelector`, which is better for Class Components.